### ISSUE TITLE
xmlns: diff too verbose

### ISSUE THREAD

**Issue by drzraf** [sflynn1812/xslt-diff#2]
### `a.xml`:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<rss version="2.0" xmlns:excerpt="http://foo/bar/1.2/excerpt/">
  <root>
    <a test="test"></a>
    <b></b>
    <c></c>
  </root>
</rss>
```

### `b.xml`:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<rss version="2.0" xmlns:excerpt="http://foo/bar/1.2/excerpt/">
  <root>
    <a test="not-this-test"></a>
    <b></b>
    <d></d>
  </root>
</rss>
```

### `output`:
```xml
<?xml version="1.0" encoding="utf-8"?>
<root xmlns:msxsl="urn:schemas-microsoft-com:xslt">
  <compare-result>
    <mismatch>
      <a xmlns:excerpt="http://foo/bar/1.2/excerpt/" test="test"/>
    </mismatch>
    <mismatch>
      <c xmlns:excerpt="http://foo/bar/1.2/excerpt/"/>
    </mismatch>
  </compare-result>
</root>
```

### `expected`:
```xml
<?xml version="1.0" encoding="utf-8"?>
<root xmlns:msxsl="urn:schemas-microsoft-com:xslt">
  <compare-result>
    <mismatch>
      <a test="test"/>
    </mismatch>
    <mismatch>
      <c />
    </mismatch>
  </compare-result>
</root>
```

### `$ xsltproc -V`
```
Using libxml 20904, libxslt 10129 and libexslt 817
xsltproc was compiled against libxml 20904, libxslt 10129 and libexslt 817
libxslt 10129 was compiled against libxml 20904
libexslt 817 was compiled against libxml 20904
```

**Comment by sflynn1812** [comment]
I'll see if I can come up with a way to remove that from the processing.

**Comment by sflynn1812** [comment]
Fixed in version 1.4
